packages/cli/src/locale/strip-locale.ts

```diff
@@ -0,0 +1,104 @@
+import { unflatten } from 'flat';
+import fs from 'fs';
+import get from 'lodash/get';
+import path from 'path';
+import { Node, Project, PropertyAssignment, SyntaxKind } from 'ts-morph';
+
+// These keys aren't directly referenced in the code base, so we add them manually here
+const whitelist = [
+  'choropleth.tested_overall',
+  'choropleth.escalation_levels',
+  'choropleth.hospital_nice',
+  'choropleth.nursing_home',
+  'choropleth.disability_care',
+  'choropleth.elderly_at_home',
+  'choropleth.sewer',
+  'choropleth.behavior',
+];
+
+const APP_LOCALE_DIR = path.join(__dirname, '../../../app/src/locale');
+
+const project = new Project({
+  tsConfigFilePath: path.join(__dirname, '../../../app/tsconfig.json'),
+});
+
+const NlOriginal = JSON.parse(
+  fs.readFileSync(path.join(APP_LOCALE_DIR, 'nl.json'), {
+    encoding: 'utf-8',
+  })
+);
+const EnOriginal = JSON.parse(
+  fs.readFileSync(path.join(APP_LOCALE_DIR, 'en.json'), {
+    encoding: 'utf-8',
+  })
+);
+
+const sourceFile = project.getSourceFile('nl.json');
+
+const propertyAssignmentNodes: PropertyAssignment[] = (
+  sourceFile?.getDescendantsOfKind(SyntaxKind.PropertyAssignment) ?? []
+)
+  // each assignment always has at least one reference (the one in the file where its written),
+  // so here we only want the ones that have more than one ref. (The ones who are referenced in some other file(s))
+  .filter((x) => x.findReferences().length > 1);
+
+const newLocaleObjects = propertyAssignmentNodes
+  .map((x) => createFullPropertyChain(x))
+  .concat(whitelist)
+  // Only keep the deepest paths:
+  .filter(
+    (x, _i, l) =>
+      l.findIndex((y) => y.startsWith(`${x}.`) && y.length > x.length + 1) ===
+      -1
+  )
+  .sort()
+  .reduce(
+    (aggr, chain) => {
+      const NlValue = get(NlOriginal, chain);
+      const EnValue = get(EnOriginal, chain);
+      if (NlValue !== undefined) {
```

Comment:
nit; I think the opposite reads a little better

```suggestion
      if (isDefined(NlValue)) {
```